takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -5,68 +5,65 @@
+      <li class="pb-2" type="disc"><a href="/about/support-services"><u>Our Data Services</u></a></li>
+      <li class="pb-2" type="disc"><a href="https://medium.com/takwimu-africa"><u>Insights &amp; Stories</u></a></li>
       <li class="pb-2" type="disc"><a href="/contact-us"><u>Contact Us</u></a></li>
       <li class="mb-0 pb-3" type="disc"><a href="/dashboard"><u>Dashboard (Admin)</u></a></li>
     </ul>
   </div>
-  <div class="col-sm-4">
-    <h5 class="pb-2">Follow us</h5>
+  <div class="col-sm-8">
+    <p class="font-weight-bold pb-0">Follow us</p>
     <div class="mt-2 d-inline-block p-2 rounded" style="background-color:#1455a0;">
-      <ul class="navbar-nav">
-        {% to_list 'twitter' 'facebook' 'instagram' 'slack' 'github' as social_medias %}
+
+        <ul class="navbar-nav">
+        {% to_list 'twitter' 'facebook' 'medium' 'github' as social_medias %}
         {% for social_media in social_medias %}
           {% if settings.takwimu.SocialMediaSetting|object_value:social_media %}
-            <li class="nav-item p-2">
-              <a class="nav-link{% if not forloop.last %} border-right {% else %} {% endif %}" 
-                href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}" target="_blank" rel="noopener">
+            <li class="nav-item">
+              <a class="nav-link float-left" href="{{ settings.takwimu.SocialMediaSetting|object_value:social_media }}"
+                target="_blank" rel="noopener">
                 <i class="fab fa-{{ social_media }} fa-fw"></i>
               </a>
+              {% if not forloop.last %}<div class="border-right m-2 float-right" style="height:24px; width:1px; opacity:0.2;"></div> {% endif %}
             </li>
           {% endif %}
         {% endfor %}
-      </ul>
+        </ul>
     </div>
-  </div>
 
-  <div class="row pt-5">
-    <div class="col-8">
-      <p class="mb-0 text-truncate" style="opacity: 0.6; font-size: 16px">Takwimu is an initiative by:</p>
-      <div class="row">
-        <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
-        </a>
-        <a href="https://codeforafrica.org" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
-        </a>
-        <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
-            class="col partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
-          <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
-        </a>
+    <div class="row pt-5">
+      <div class="col-12">
```

Comment:
`.row` and `.col-12` mean it's a normal `<div/>` block. Please remove one of them.